tests/integration/test_sign_in.py

```diff
@@ -31,7 +31,7 @@ class TestSignIn(unittest.TestCase):
 
     def setUp(self):
         self.app = app.test_client()
-        self.oauth_token = {
+        self.auth_token = {
             "id": 1,
             "access_token": "8c77e013-d8dc-472c-b4d3-d4fbe21f80e7",
```

Comment:
This bit needs to be cleaned up to better reflect what you actually get these days.  The old django-oauth2-test would give you an access token (and I think expiry?).  These days the 'token' is just an empty dict I think (but double check before you start deleting stuff!)

Reply:
Auth hands back a response object here, as a dict, and we need to avoid confusion with JWTs. Removed the expired OAUTH token from the tests ;  renamed remaining ".auth_token" to ".auth_response"; set the initial ".auth_response" to an empty dict.